src/ruby/yast/rspec.rb

```diff
@@ -1,6 +1,7 @@
+require "yast/rspec/helpers"
+require "yast/rspec/matchers"
 require "yast/rspec/scr"
 require "yast/rspec/shortcuts"
-require "yast/rspec/matchers"
 
 RSpec.configure do |c|
   c.include Yast::RSpec::Shortcuts
```

Comment:
The [Rspec include](https://rubydoc.info/github/rspec/rspec-core/RSpec/Core/Configuration#include-instance_method) adds the methods into the example groups.

For mocking YaST modules that is usually too late, you need to mock them *before* loading any test as the `Yast.import` is called very early. IIRC we mock all YaST modules in the `test_helper.rb` so the include would not make sense, it would be unused.